src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -129,8 +129,9 @@ class TransferBalancePage extends React.Component {
             ? CONST.WALLET.TRANSFER_METHOD_TYPE.ACH
             : CONST.WALLET.TRANSFER_METHOD_TYPE.INSTANT;
 
-        const transferAmount = PaymentUtils.subtractWalletTransferBalanceFee(this.props.userWallet.currentBalance).toFixed(2);
-        const canTransfer = transferAmount > CONST.WALLET.TRANSFER_BALANCE_FEE;
+        const calculatedFee = PaymentUtils.calculateWalletTransferBalanceFee(this.props.userWallet.currentBalance, selectedPaymentType);
+        const transferAmount = this.props.userWallet.currentBalance - calculatedFee;
+        const canTransfer = transferAmount > calculatedFee;
```

Comment:
Shouldn't this line either be `this.props.userWallet.currentBalance > calculatedFee` or `transferAmount > 0`? 

As it is, you couldn't transfer $0.50 because `50-25 = 25` and `25 !> 25`

Reply:
Yeah, it is supposed to be `transferAmount > 0`. Thanks for checking in.